### ISSUE TITLE
1.16.1 - Crash at 100% world creation

### ISSUE THREAD

**Issue by bessiq** [Lupicus/NastyMobs#1]
NastyMobs-1.16.1-1.0.0.3.jar
Forge 32.0.47

[crash-2020-07-04_20.04.02-server.txt](https://github.com/Lupicus/NastyMobs/files/4874216/crash-2020-07-04_20.04.02-server.txt)

Looking forward to testing out these nasty mobs, but when any new world I create reaches 100%, the game crashes. The crash report seems to indicate this mod, and disabling it allows the world to load normally. But let me know if you think it might be something else, or just need more info. Thanks!

**Comment by Lupicus** [comment]
Okay thanks. I'll take a look. It did run on older version of 1.16, so something changed since forge 32.0.25 or it is problem with another mod maybe.

**Comment by Lupicus** [comment]
I ran the mod with both kinds of 32.0.47 servers without a problem, so it is probably issue with another mod.

**Comment by bessiq** [comment]
Thanks for looking into it.

**Comment by Lupicus** [comment]
If this is a modpack then what is it name? Though I'm deciding on code to prevent this.

**Comment by bessiq** [comment]
It's not a modpack (at least not yet). It's just a personal pack I've been playing with to try out 1.16 mods. I've changed mods and configs many times since I tried Nasty Mobs, so I just re-added it. Unfortunately, the same thing happens.

Forge: 32.0.66
NastyMobs-1.16.1-1.0.0.3.jar

Crash report: https://www.dropbox.com/s/4mt5lfue0akl6gd/crash-2020-07-16_21.19.53-server.txt?dl=0
latest.log: https://www.dropbox.com/s/wd10kheoj4qaf07/latest.log?dl=0
debug.log: https://www.dropbox.com/s/lqcyceuaymxjy7w/debug.log?dl=0
